styles/morning.json

```diff
@@ -0,0 +1,338 @@
+						},
+						"typography": {
+							"textDecoration": "underline"
+						},
+						":hover": {
+							"typography": {
+								"textDecoration": "none"
+							}
+						}
+					}
+				}
+			},
+			"core/post-terms": {
+				"typography": {
+					"fontWeight": "400",
+					"fontSize": "var:preset|font-size|medium"
+				},
+				"color": {
+					"text": "var:preset|color|contrast"
+				},
+				"elements": {
+					"link": {
+						"color": {
+							"text": "var:preset|color|contrast"
+						},
+						"typography": {
+							"textDecoration": "underline"
+						},
+						":hover": {
+							"typography": {
+								"textDecoration": "none"
+							}
+						}
+					}
+				}
+			},
+			"core/post-title": {
+				"typography": {
+					"fontWeight": "900",
+					"letterSpacing": "-0.96px"
+				},
+				"elements": {
+					"link": {
+						"color": {
+							"text": "var:preset|color|contrast"
+						}
+					}
+				}
+			},
+			"core/pullquote": {
+				"color": {
+					"text": "var:preset|color|contrast"
+				},
+				"typography": {
+					"fontSize": "var:preset|font-size|x-large",
+					"letterSpacing": "-0.76px"
+				},
+				"elements": {
+					"cite": {
+						"typography": {
+							"fontSize": "1.12rem",
+							"letterSpacing": "-0.14px"
+						},
+						"color": {
+							"text": "var:preset|color|primary"
+						}
+					}
+				}
+			},
+			"core/quote": {
+				"color": {
+					"text": "var:preset|color|contrast"
+				},
+				"typography": {
+					"fontSize": "1.5rem",
+					"fontWeight": "500",
+					"letterSpacing": "-0.18px"
+				},
+				"elements": {
+					"cite": {
+						"typography": {
+							"fontSize": "1.12rem",
+							"fontWeight": "300",
+							"letterSpacing": "-0.14px"
+						},
+						"color": {
+							"text": "var:preset|color|primary"
+						}
+					}
+				}
+			},
+			"core/site-tagline": {
+				"typography": {
+					"fontSize": "var:preset|font-size|large"
+				}
+			},
+			"core/site-title": {
+				"typography": {
+					"fontFamily": "var:preset|font-family|ysabeau-office",
+					"fontSize": "var:preset|font-size|large",
+					"fontWeight": "700",
+					"textTransform": "uppercase"
+				}
+			}
+		},
+		"elements": {
+			"button": {
+				"typography": {
+					"fontFamily": "var:preset|font-family|literata",
+					"fontSize": "1.375rem",
+					"fontWeight": "900",
+					"letterSpacing": "-0.36px"
+				},
+				"color": {
+					"text": "var:preset|color|contrast",
+					"background": "var:preset|color|accent-1"
+				},
+				"border":{
+					"color": "var:preset|color|accent-1",
+					"radius": "0px"
+				}
+			},
+			"heading": {
+				"color": {
+					"text": "var:preset|color|contrast"
+				},
+				"typography": {
+					"fontFamily": "var:preset|font-family|literata",
+					"fontWeight": "900"
+				}
+			},
+			"link": {
+				"color": {
+					"text": "var:preset|color|contrast"
+				}
+			}
+		},
+		"variations": {
+			"post-terms-1": {
+				"typography": {
+					"fontSize": "16px"
```

Comment:
Does this need to be hardcoded? Is there no matching responsive preset size?

Reply:
There's no preset for 16px, which is the size of the font size for this particular element in the variation. We have `--wp--preset--font-size--normal` from Gutenberg, which is 16px, but we can't use that.

If we don't want to hardcode the value, we could use medium (18px) or small (14px) alternatives.

Reply:
I'd always go for the larger :)

Reply:
Should we run this by Bea once she's back?